File: Valid Anagram.java
//242. Valid Anagram Leetcode
class Solution {
    public boolean isAnagram(String s, String t) 
    {
        if(s.length()!=t.length())
            return false;
        int[] values1=new int[26];
        int[] values2=new int[26];
        char[] a=s.toCharArray();
        char[] b=t.toCharArray();
        for(int i=0;i<a.length;i++)
        {
            values1[a[i]-97]++;
            values2[b[i]-97]++;
        }
        for(int i=0;i<values1.length;i++)
        {
            if(values1[i]!=values2[i])
                return false;
        }
        return true;
    }
}

File: Valid Anagram.py
#242. Valid Anagram LeetCode
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        lst1=[element for element in s]
        lst2=[element for element in t]
        lst1.sort()
        lst2.sort()
        if(lst1!=lst2):
            return False
        return True
